training/deepspeed/Containerfile

```diff
@@ -0,0 +1,17 @@
+# Containerfile for running deepspeed training 
+FROM nvidia/cuda:12.1.1-cudnn8-devel-ubi9
+
+RUN dnf install -y python python-devel git
+RUN python -m ensurepip --upgrade
+RUN pip3 install torch==2.1.2 --index-url https://download.pytorch.org/whl/cu121
```

Comment:
I feel like all of this could be a `requirements.txt` that you add to the container, is there a reason to do it this way instead?

Reply:
updated, thanks!